bin/run_fs_recon.py

```diff
@@ -122,10 +121,11 @@ def _run(subjects_dir, subject, raw_dir, force, mp, volume):
     run_subprocess(['mri_concat', '--rms', '--i', mri,
                     '--o', op.join(subjects_dir, subject, 'mri/orig/001.mgz')],
                    env=this_env)
-    run_subprocess(['recon-all', '-openmp', mp, '-subject', subject, '-all'], env=this_env)
+    run_subprocess(['recon-all', '-openmp', '%.0f' % mp, '-subject', subject, '-all'], env=this_env)
```

Comment:
FYI if you want to force an integer you can do `'%d' % mp`. But since you know it's an integer you can also just do `str(mp)`. But this should work too